test/JDBC/input/BABEL-4926.sql

```diff
@@ -0,0 +1,158 @@
+-- create a new table BABEL_4926 1
+create table BABEL_4926(babel_4926_COL int);
+go
+select babel_4926_COL from BABEL_4926;
+go
+-- Create a new table BABEL_4926_REPO with column babel_4926_COL 
+select babel_4926_COL into BABEL_4926_REPO from BABEL_4926;
+go
+-- Indexing over column babel_4926_COL
+create index IDX_REPRODUCTION on BABEL_4926_REPO(babel_4926_COL);
+go
+-- output is the lowercase name,original_name
+select attname,attoptions from pg_attribute where attname like '%babel_4926%' and attrelid = (select oid from pg_class where relname = 'babel_4926_repo') order by attname asc;
+go
+-- dropping the table BABEL_4926
+drop table BABEL_4926;
+go
+-- dropping the table BABEL_4926_REPO
+drop table BABEL_4926_REPO;
+go
+
+
+
```

Comment:
Refactor: Try to group tests as per the case, this makes it easier to review and check the edge cases.